python/src/UniformMuSigma_doc.i.in

```diff
@@ -1,13 +1,17 @@
 %feature("docstring") OT::UniformMuSigma
 "Uniform distribution parameters.
-
+    
 Parameters
 ----------
 mu : float
     Mean.
+
+    Default value is 0.0.
 sigma : float
     Standard deviation :math:`\sigma > 0`.
 
+    Default value is 0.57735.
```

Comment:
What is the "0.57735" magic number?

Reply:
probably sqrt(1/3)

Reply:
ok. I had not documented correctly... in the code, the expression is exact. I modified the doc and explained the reason (the values are define to get the default values of the native parameters)